src/network/bmproto.py

```diff
@@ -422,16 +430,16 @@ def _decode_addr(self):
 
     def bm_command_addr(self):
         """Incoming addresses, process them"""
-        addresses = self._decode_addr()
+        addresses = self._decode_addr()      # pylint: disable=redefined-outer-name
         for i in addresses:
             seenTime, stream, services, ip, port = i
             decodedIP = protocol.checkIPAddress(str(ip))
             if stream not in state.streamsInWhichIAmParticipating:
                 continue
             if (
-                decodedIP and time.time() - seenTime > 0 and
-                seenTime > time.time() - BMProto.addressAlive and
-                port > 0
+                    decodedIP and time.time() - seenTime > 0 and
+                    seenTime > time.time() - BMProto.addressAlive and
+                    port > 0
```

Comment:
I was trying to save horizontal space and this should not violate PEP8 or any other style guide...

Reply:
So why 996e71a was merged without any warning?

Reply:
I think github check didn't run correctly.

Reply:
I think it may be a bug with the checks. I'm getting the same error locally.

Reply:
This is really a bug: https://github.com/PyCQA/pylint/issues/289

Reply:
For the checks GitHub App, pylint 1.6.5 is used, whereas locally I have 1.9.2. I don't want to change anything as the server seems to misbehave and there is no failover. We'll have to wait until I can put the new system using jenkins into production. I hope it will be by the end of this month.

Reply:
I would say we leave these pylint problems for later after the checks are upgraded.

Reply:
I meant we could do something like
```diff
diff --git a/setup.cfg b/setup.cfg
index 3236eed1..68a8181c 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -4,6 +4,10 @@
 [pycodestyle]
 max-line-length = 119
 
+[pylint.messages control]
+disable = C0330
+# C0330 - wrong-hanging-indentation
+
 [flake8]
 max-line-length = 119
 ignore = E722,F841
```

and invoke pylint with `--rcfile setup.cfg` because if you are going to format the code according to pylint bugs it will become unreadable soon.